scene_select/do_ard.py

```diff
@@ -113,13 +113,14 @@ def do_ard(
     This function assumes a l1 zip file has been written to the jobdir.
     Though if you specify l1_zips in a list, and usgs_level1_files is None,
       it will write the file."""
+    LOGGER.info("do_ard", **locals())
     try:
         _calc_node_with_defaults(ard_click_params, l1_count)
     except ValueError as err:
         print(err.args)
         LOGGER.warning("ValueError", message=err.args)
 
-    if l1_zips is not None and len(l1_zips) > 0:
+    if l1_zips is not None and len(l1_zips) >= 0:
```

Comment:
i think it would be better we do a truth test as it's more accurate.
I recommend `if l1_zips  and len(l1_zips) >= 0:`

From the change in line 125 in scene_select/ard_reprocessed_l1s.py, I also have a question. Do we want to test l1_zips is an empty string (ie. "")?

```
>>> j=""
>>> dd = len(j)
>>> dd
0
```

Reply:
I want this to write if there is an empty string, so `if l1_zips  and len(l1_zips) >= 0:` will not work.

Reply:
I'm testing dropping it back to `if l1_zips is not None:`

Reply:
oh, ok. that makes sense :)

Reply:
no. look at it more as don't write if l1_zips is None. if [] or [1, 2], write.